API_outline.md

```diff
@@ -223,4 +223,53 @@ Gets a user by ID
 }
 ```
 
-The `error` key can be one of `BAD_REQUEST` or `USER_NOT_FOUND`
\ No newline at end of file
+The `error` key can be one of `BAD_REQUEST` or `USER_NOT_FOUND`
+
+## POST /api/register
+
+INPUT FORMAT
+
+```json
+{
+  "name": "john doe",
+  "email": "johndoe@gmail.com",
+  "phone_number": "8312345678",
+  "password": "johndoe",
+}
+```
+### Example response
+```json
+{
+  "message": "REGISTERED SUCCESFULLY"
+}
+
+```
+### Example error response
+
+```json
+{
+  "error": "REGISTERED SUCCESFULLY"
```

Comment:
Should this be here? It looks like a success message under the "Example error response" section